Resources/Prototypes/Entities/Clothing/Masks/masks.yml

```
@@ -291,9 +291,9 @@
   description: A sterile mask designed to help prevent the spread of diseases.
   components:
   - type: Sprite
-    sprite: Clothing/Mask/sterile.rsi
+    sprite: _Harmony/Clothing/Mask/sterile.rsi
   - type: Clothing
-    sprite: Clothing/Mask/sterile.rsi
+    sprite: _Harmony/Clothing/Mask/sterile.rsi
```

Needs a harmony comment
```suggestion
    sprite: _Harmony/Clothing/Mask/sterile.rsi // Harmony
```